### PR TITLE
Update Package Service

### PR DESCRIPTION
**Pull request by ckittel**
* Move from NodeJS 8.12 to Node 19.9
* Move from TypeScript 3.8 to 5.0
* Full npm package update
* Gulp to ESModule structure
* Fixed an issue where ./up.sh was referencing the wrong image name

Unit tests:
4cca5cb2-1de7-4707-b67d-bc16c4529aed

readme.md instructions:
2b215398-77b3-4f69-a00f-d14dca0985a4

Logging (from within container):
2bc0f3f3-2c82-401f-b328-9823a30d8e97

### CODE REVIEW

File: src/shipping/package/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM node:8.12.0-alpine as base
```
Comment (ferantivero) [inline src/shipping/package/Dockerfile:1]: just out of curiosity, and if and only if you know the answer (not asking for a change or re-test anything here), was this change really required?

File: src/shipping/package/app/models/package.ts [hunk 2]
```diff
@@ -15,5 +16,8 @@ export class Package {
 
     constructor(id? : string) {
         this._id = id || (new ObjectID()).toHexString();
+        this.tag = '';
```
Comment (ferantivero) [inline src/shipping/package/app/models/package.ts:19]: look like the type class already has initilizers for `tag`, `weight`, `size`, do we want to re-initialize them in here?

**Review by ferantivero — COMMENTED** [review]
nicely done @ckittel, LGTM just a few question for my own knowledge.

File: src/shipping/package/Dockerfile (hunk 1 above)
Comment (ckittel) [reply src/shipping/package/Dockerfile:1]: I honestly don't know exactly what of all these changes are required.  I just simply went with the near-latest of everything and things started working.  This was built with node 19, but no idea if it was the node version, the TS version, etc that really make it all work again.

File: src/shipping/package/app/models/package.ts (hunk 2 above)
Comment (ckittel) [reply src/shipping/package/app/models/package.ts:19]: The compiler either had me add them on the members themselves or in the constructor.  I added them on the members for the class because there was no constructor.  Since there was a constructor here, added them with the constructor.

**Review by ferantivero — APPROVED** [review]
LGTM, please let's go ahead and merge while we discuss from the another review. 🚢
